rollup.config.js

```diff
@@ -15,7 +15,7 @@ const createConfig = (pkgName) => ({
             presets,
             plugins,
             exclude: "node_modules/**",
-        }),      
```

Comment:
??? is prettier being run differently somehow?

Reply:
Not sure.  The extra whitespace was added in https://github.com/Khan/wonder-blocks/commit/c4aa5a633effcf32b86c83a669c182127afec80a.